File: Kritika Tripathi/Basics Of Programming/Patterns/Pattern16.cpp
#include <iostream>
using namespace std;
int main(){
    int n;
    cin >> n;
    int j,l;
    //write your code here
    for(int i=1; i<=n; i++){
        for(j=1; j<=i; j++){
            cout<<j<<"\t";
        }
        for(;j<=i+2*(n-i)-1; j++){
            cout<<"\t";
        }
        if(i==n){ 
            l=i-1;
        }
        else l=i;
        for(;j<=2*n-1; j++,l--){
            cout<<l<<"\t";
        }
        cout<<endl;
    }
    
}

File: V.Vennela/Basics of Programming/Patterns/Pattern16.java
import java.util.*;

public class Pattern16{

public static void main(String[] args) {
    Scanner scn = new Scanner(System.in);

     // write ur code here
     int n = scn.nextInt();
     int x=0;
     for(int i=1;i<=n;i++)
     {
         for(int j=1;j<=2*n-1;j++)
         {
             if(j>n)
            {
                x=(2*n)%j;
            }
            else
            {
                x=j;
            }
             if(x>i)
                System.out.print("\t");
             else
                System.out.print(x+"\t");
         }
         System.out.println();
     }

 }
}
